source/extensions/common/dynamic_forward_proxy/dns_cache.h

```diff
@@ -148,6 +179,16 @@ class DnsCache {
    * @return all hosts currently stored in the cache.
    */
   virtual absl::flat_hash_map<std::string, DnsHostInfoSharedPtr> hosts() PURE;
+
+  /**
+   * increment overflow counter.
+   */
+  virtual void dnsCacheStatsOverflowInc() PURE;
```

Comment:
Not sure why this is still here.

Reply:
I think that there is no way to access stats in current implementation. So this may be required...

Reply:
Sorry my point is you don't need to externally access the stats, it's an abstraction leak. Everything should be handled internally to the DNS cache impl.

/wait

Reply:
done